### PR TITLE
Adds some new Nightmare sprites and a description to the light eater

### PR DESCRIPTION
**Pull request by ghost**
Adds some new sprites to the Nightmare race based on the old shadowling ones, and makes the light eater into spooky tentacles instead of an arm blade (plus gave it some attack verbs and a desc.)

I felt like a new side-antag using recycled assets from shadowpeople etc. looked and felt weird, so I whipped up some sprites. They got missed in the initial PR, so I re-PRed them here. ~~We also all knew that I'd make changes to nightmares.~~

@KorPhaeron You're the design gamer for nightmares, so whether or not this PR stays is up to you.

### CODE REVIEW

**Comment by nicbn** [comment]
Nightmares remember me [SCP 106](http://www.scp-wiki.net/scp-106)... That's awesome!

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 1]
```diff
@@ -70,15 +69,17 @@
...
+	icon_state = "tentacle"
+	item_state = "light_eater"
 	force = 25
 	armour_penetration = 35
 	lefthand_file = 'icons/mob/inhands/antag/changeling_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
 	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
 	w_class = WEIGHT_CLASS_HUGE
 	sharpness = IS_SHARP
+	attack_verb = list("consumed", "devoured", "eaten")
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: these don't actually make sense as attack verbs for something that isn't an instakill

Comment (DoctoreMad) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Eaten is the only one that will make no sense grammatically. Everything else can be chalked to eating/devouring only parts of the target.

Comment (Xhuis) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: It's named "light eater" so my headcanon is that (with the tentacly look) it bites off chunks of the target, yeah; imagine a squid with a toothy mouth on the inside of the tentacles

Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: It turns everything into dust though

Comment (Xhuis) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Spooky eldritch magic

**Comment by Repukan** [comment]
Pictures please

**Comment by FuryMcFlurry** [comment]
image

Hows this

**Comment by MoreRobustThanYou** [comment]
@nicbn fuck you for making me look at the pic on that page again

**Comment by KorPhaeron** [comment]
My initial reaction is dislike but I will think about it

**Comment by WJohn** [comment]
Pretty sprites. 👍

**Comment by nicbn** [comment]
@MoreRobustThanYou 
**THAT PIC?**

**Comment by Militaires** [comment]
IM STILL IN A BREEM LIGHT EATER

**Comment by Xhuis** [comment]
Is this likely enough to get merged that I should fix its conflicts?

**Comment by MoreRobustThanYou** [comment]
@Xhuis ye

**Comment by optimumtact** [comment]
I'm gonna assume kor's silence means no.

they can show up later and contradict me or have you make a new pr

**Comment by Xhuis** [comment]
F

**Pull request closed without merging**